src/core/Form/TextInput/TextInput.baseStyles.tsx

```diff
@@ -69,9 +69,12 @@ export const baseStyles = withSuomifiTheme(
       width: 100%;
       min-height: 40px;
       padding-left: ${theme.spacing.insetL};
+      border-color: ${theme.colors.depthDark3};
 
-      &::placeholder {
```

Comment:
Was this a bug or is it now wrong in both TextInput and SearchInput?

Reply:
Probably just missing the '&'. Works both ways. Probably ended up like this by accident since the values below had this syntax. I'll add the '&' though.

Reply:
I'd make sure that it doesn't affect the output. It's a bit tricky with Sass style compositions sometimes.

Reply:
_I had some problems myself some time ago with some browser when I was missing the `&` from the beginning._

Reply:
Maybe it should be added to other pseudos as well?

Reply:
Now applied to all pseudos in basestyles files.